Source/Widgets/Geocoder/GeocoderViewModel.js

```diff
@@ -80,12 +80,12 @@ define([
         });
 
         this._searchCommand = createCommand(function() {
-            that.hideSuggestions();
             that._focusTextbox = false;
             if (defined(that._selectedSuggestion)) {
                 that.activateSuggestion(that._selectedSuggestion);
                 return false;
             }
+            that.hideSuggestions();
```

Comment:
Was originally concerned about moving this after an early return, but, `activateSuggestion` calls this already, so this is actually the correct location for this.